roles/contiv/handlers/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: reload systemd
+  command: systemctl --system daemon-reload
+
+- name: restart netmaster
+  service: name=netmaster state=restarted
+  when: netmaster_started.changed == false
+
+- name: restart netplugin
+  service: name=netplugin state=restarted
+  when: netplugin_started.changed == false
```

Comment:
@DanyC97 

I agree in principle but even the different roles within Openshift itself are not consistent with each other. Compare openshift_cli with openshift_loadbalancer or many other such cases which are different from each other in style/ formatting.

Note that contiv also needs to maintain its own style consistency between what we deliver for Openshift versus other container platforms.


I will make the changes to netmaster.yml and netplugin.yml to be consistent within themselves but we can keep going forever if we want to match with all the different Openshift files as well as all vendor submitted files to have the same formatting style.